### ISSUE TITLE
bug: Foundry or system is disabling Tidy sheet buttons wholesale, preventing tab navigation for Locked Compendium sheets

### ISSUE THREAD

**Issue by kgar** [kgar/foundry-vtt-tidy-5e-sheets#168]
Potentially either Foundry or dnd5e seems to be disabling all of the buttons on my sheets when opening a sheet from a locked compendium.

As a result, viewing content from the compendium disables tab navigation among other features.

![image](https://github.com/kgar/foundry-vtt-tidy-5e-sheets/assets/4632129/8e0fa12e-a33d-434c-9d57-cf828b818856)

**Comment by kgar** [comment]
It is part of `FormApplication`:
```js
  /**
   * If the form is not editable, disable its input fields
   * @param {HTMLElement} form    The form HTML
   * @protected
   */
  _disableFields(form) {
    const inputs = ["INPUT", "SELECT", "TEXTAREA", "BUTTON"];
    for ( let i of inputs ) {
      for ( let el of form.getElementsByTagName(i) ) {
        if ( i === "TEXTAREA" ) el.readOnly = true;
        else el.disabled = true;
      }
    }
  }
```

The way forward is to override and NOOP this on Tidy sheets and selectively disable where inputs/buttons/etc. may be found in the application, based on the `isEditable` prop. Ideally, we should be able to greatly simplify a lot of content if we lean more heavily on the `editable` / `isEditable` data.

This will require a bit of a refactor, as I have already been leveraging the `editable` context prop without this core logic in mind.

From a sheets design standpoint, since I am intentionally departing from the full `FormApplication` / `DocumentSheet` approach, having my logic clearly displayed on the templates and components for how things disable should do a long way to making the sheets more self-documenting.

**Comment by kgar** [comment]
This has been fixed in 0.1.22.
